src/schema/v2/__tests__/searchCriteriaLabel.test.ts

```diff
@@ -73,114 +73,125 @@ describe("resolveSearchCriteriaLabels", () => {
     ])
   })
 
-  it("formats medium criteria", async () => {
-    const parent = {
-      additionalGeneIDs: [
-        "painting",
-        "photography",
-        "sculpture",
-        "prints",
-        "work-on-paper",
-        "nft",
-        "design",
-        "drawing",
-        "installation",
-        "film-slash-video",
-        "jewelry",
-        "performance-art",
-        "reproduction",
-        "ephemera-or-merchandise",
-      ],
-    }
+  describe("medium criteria", () => {
+    it("formats medium criteria", async () => {
+      const parent = {
+        additionalGeneIDs: [
+          "painting",
+          "photography",
+          "sculpture",
+          "prints",
+          "work-on-paper",
+          "nft",
+          "design",
+          "installation",
+          "film-slash-video",
+          "jewelry",
+          "performance-art",
+          "reproduction",
+          "ephemera-or-merchandise",
+        ],
+      }
 
-    const labels = await resolveSearchCriteriaLabels(parent, _, _, _)
+      const labels = await resolveSearchCriteriaLabels(parent, _, _, _)
 
-    expect(labels).toIncludeAllMembers([
-      {
-        name: "Medium",
-        displayValue: "Painting",
-        value: "painting",
-        field: "additionalGeneIDs",
-      },
-      {
-        name: "Medium",
-        displayValue: "Photography",
-        value: "photography",
-        field: "additionalGeneIDs",
-      },
-      {
-        name: "Medium",
-        displayValue: "Sculpture",
-        value: "sculpture",
-        field: "additionalGeneIDs",
-      },
-      {
-        name: "Medium",
-        displayValue: "Prints",
-        value: "prints",
-        field: "additionalGeneIDs",
-      },
-      {
-        name: "Medium",
-        displayValue: "Work on Paper",
-        value: "work-on-paper",
-        field: "additionalGeneIDs",
-      },
-      {
-        name: "Medium",
-        displayValue: "NFT",
-        value: "nft",
-        field: "additionalGeneIDs",
-      },
-      {
-        name: "Medium",
-        displayValue: "Design",
-        value: "design",
-        field: "additionalGeneIDs",
-      },
-      {
-        name: "Medium",
-        displayValue: "Drawing",
-        value: "drawing",
-        field: "additionalGeneIDs",
-      },
```

Comment:
Good thought to consolidate the mappings — just noting that this means:

- the pills for Alerts will now match the medium labels at the _artwork_ level rather than what's exposed in _artwork grid_ medium filters, i.e.

  <img width="3008" alt="remap" src="https://user-images.githubusercontent.com/140521/221242863-bbd0da16-7ce8-412d-a35b-264e320f5d50.png">

- **Drawing** (which overlaps with **Works on Paper**) will go from having first-class support to being treated as an "other" medium

If we're okay with sometimes having more verbose labels and "demoting" Drawings somewhat then I'm fine with this change

Reply:
Is there any harm is defining a new "Drawings" entry in the mediumTypes mapping? That way, if an alert is created from a filtered artist grid, the alert pill should still show up as "Drawing" and map to the [drawing](https://www.artsy.net/gene/drawing) gene (vs [work-on-paper](https://www.artsy.net/gene/work-on-paper)).

Reply:
Meaning [this](https://github.com/artsy/metaphysics/blob/main/src/lib/artworkMediums.ts)? I don't see the harm in that change, other than diluting the meaning of that mapping (since it would no longer be a list of CMS/artwork medium types.

The change in https://github.com/artsy/metaphysics/pull/4826/commits/0d39ff3677caef461f728fd6bfd39963a1331033 is an alternative approach, right? That adds another corner to this logic — but that seems hard to avoid given the medium vs gene indirection.

Seems like a lot of imperfect choices, and I don't have a strong opinion. Could merge this as-is currently.

Reply:
Sounds good. I'll merge this PR as is. I think it's a reasonable compromise for now to extend the CMS/filter mapping and then add an Other fallback.